src/core/Akka.Streams/Implementation/Fusing/Ops.cs

```diff
@@ -2512,61 +2513,47 @@ public Expand(Func<TIn, IEnumerator<TOut>> extrapolate)
         /// </returns>
         public override string ToString() => "Expand";
     }
+    
+    #nullable enable
```

Comment:
Enable `nullable` to take advantage of the changes made in https://github.com/akkadotnet/akka.net/pull/7520